modules/flowable-http/src/test/java/org/flowable/http/bpmn/HttpServiceTaskTest.java

```diff
@@ -497,6 +500,31 @@ public void testDeleteResponseEmptyBody() {
         assertNull(runtimeService.getVariable(processInstanceId, "myResponse"));
     }
 
+    @Test
+    @Deployment
+    public void testGetWithVariableNameAndSkipExpression() {
+        Map<String, Object> variables = new HashMap<>();
+        variables.put("_ACTIVITI_SKIP_EXPRESSION_ENABLED", true);
```

Comment:
can you change this to _FLOWABLE_SKIP_EXPRESSION_ENABLED, and the same for the other lines using this variable?

Reply:
Updated